.travis.yml

```diff
@@ -2,35 +2,4 @@ language: java
 
 notifications:
   email:
-    - mgrigorov@apache.org
-  irc: 
-    channels:
-      - "chat.freenode.net##wicket"
-    template:
-      - "%{repository}, %{author}, (%{commit}) : %{message} "
-      - "Build details: %{build_url}"
-      - "Duration: %{duration}"
-
-env:
-  global:
-    - secure: "I0PbW4iQlyMO/yztGnZ/qiaMm7IS51QqAGhA5HQYKVh/ywNpQpVrpZaFRyf8OlBAemqKmOTp3EcvR225hb6RQCFilkN2vCMbMO/b7BbMRVhfV6PlpkfiuP2EehgT5vXl74tF/WbnijbvLpQppLPqbC0yYKOVplVcNEDs0LcVLfk="
-    - secure: "PKgEV+4JuiCQ+69lYz8kDXLVKxn107EzGd+jq0BPxZK09LqMsvQpvkF0EZGF0gk1xXt3Gn2itVeKPbcxgufccuzVNoQnnXljvKn9+6ITTKg2HZA7UP4MrdOhhw1m+GMRKYVEdPz6qcG4FXlBfw3zgrmiZZ89K6mggPFeyf1VFDw="
-
-cache:
-  directories:
-    - $HOME/.m2
-
-install: true
-
-script: 
-  - 'MVN_OPTS="-Dmaven.javadoc.skip=true -Dsource.skip=true -B -V --quiet"'
-  - "DEPLOY_OR_TEST=test"
-  - '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && DEPLOY_OR_TEST=deploy'
-  - "echo DEPLOY_OR_TEST=$DEPLOY_OR_TEST"
-  - "jdk_switcher use openjdk6"
-  - "mvn -s sonatype-settings.xml clean $DEPLOY_OR_TEST -N $MVN_OPTS"
-  - "cd jdk-1.6-parent"
-  - "mvn -s ../sonatype-settings.xml clean $DEPLOY_OR_TEST $MVN_OPTS"
-  - "cd ../jdk-1.7-parent"
-  - "jdk_switcher use oraclejdk7"
-  - "mvn -s ../sonatype-settings.xml clean $DEPLOY_OR_TEST $MVN_OPTS"
+    - twitch@nervestaple.com
```

Comment:
Another way is to use `git rebase --interactive THE_COMMIT_BEFORE_YOUR_CHANGES` and then delete the line of the commit that touched .travis.yml. It will work only if this commit doesn't touch other files. Otherwise you'd lose the meaningful changes too.